### ISSUE TITLE
[MAS1.3.1][Info and Relationships- Open Bot Dialog] Placeholder and label of the “Bot URL” edit field are same.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1955]
**Actual Result:**
In "Open Bot Dialog, the placeholder and the label of "Bot URL" edit field are same which is also verified from the code.

**Expected Result:**
The placeholder should be different from the label. The purpose of the placeholder is to provide the additional information about the edit field, so it should be more descriptive than the label.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to the Welcome tab and reach at "Open Bot" button and activate it.
4. "Open a Bot" dialog gets appeared, navigate to "Bot URL" edit field.
5. Observe the issue.

**User Impact:**
User gets confused if the placeholder and label are same.

**Observation:**
If the placeholder and the label are same the voiceover, will also narrates the same information twice which misguide the voiceover dependent user.

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment:**
![Placeholder and label of the “Bot URL” edit field are same](https://user-images.githubusercontent.com/55279608/67927063-a5d42a00-fbdd-11e9-8891-93a10634f61f.png)

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is fixed now. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0-94878
